src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1285,6 +1291,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 			block.frequencyOnChunk = 7;
 			block.minGenerateHeight = 21;
 			block.maxGenerateHeight = 92;
+			block.isBonemealable = !emptyLists;
 			if (!emptyLists) {

```

Comment:
Why not go the same way here?
```suggestion
			if (!emptyLists) {
				plant.isBonemealable = true;
```

Reply:
Good point